dmsnitch/resource_snitch.go

```diff
@@ -95,6 +110,7 @@ func newSnitchFromResource(d *schema.ResourceData) *Snitch {
 		Status:   d.Get("status").(string),
 		Interval: d.Get("interval").(string),
 		Type:     d.Get("type").(string),
+		Email:    email,
```

Comment:
```suggestion
		AlertEmails:    emails,
```
Same change again here